sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushTest.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.appcenter.codepush;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CodePushTest {
+    @Test
```

Comment:
please add blank line before annotations such as @Test or @Override, also no underscore in method names, use camelCase

Reply:
@guperrot Reformatted, thanks.